### PR TITLE
Adds February 2021 newsletter

### PR DESCRIPTION
**Pull request by jbteves**
Adds February 2021 newsletter and ultra-short script for consistent git history formatting

### CODE REVIEW

**Comment by jbteves** [comment]
@ME-ICA/tedana-devs please give this a look if you have the time.
For those who missed it, we've decided to create this repository to store newsletters and make the review process for newsletters more in line with our other review processes.

**Comment by tsalo** [comment]
Long term, we might want a CI step that renders the HTML for easy viewing.

File: newsletter-2021-02.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<h1>tedana February 2021 Update</h1>
+Since our late December newsletter, we have created a new stable release available on PyPI!
+You can upgrade your local installation to the latest 0.0.9 version by running <xmp>pip install -U tedana</xmp>.
```
Comment (tsalo) [inline newsletter-2021-02.html]: At least on Chrome, after downloading the raw HTML file, `<xmp>pip install -U tedana</xmp>` ends up on its own line, and the following period shows up before "We encourage" on its own line.

I don't do much with HTML, so maybe this isn't an issue?

File: newsletter-2021-02.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+<h2>New Contributors</h2>
+We are excited to welcome three new contributors since our last newsletter: M. Versluis, Z. Ahmed, and A. Khojandi!
+Thanks to all of our contributors for their continued input and help in creating tedana.
+The community has now grown to 20 contributors, and we always look forward to seeing more new faces!
+If you’re not sure where to start, please feel free to open an issue or join our next
+<a href="https://tedana.readthedocs.io/en/latest/developing.html?highlight=developer%20call#monthly-developer-calls">monthly developer's call</a>.
+
+<h2>Governance Updates</h2>
+We have recently overhauled our governance structure, moving from a Benevolent-Dictator-for-Life (BDFL) model to a Steering committee structure.
```
Comment (tsalo) [inline newsletter-2021-02.html]:
```suggestion
We have recently overhauled our governance structure, moving from a Benevolent-Dictator-for-Life (BDFL) model to a steering committee structure.
```

Or "Steering Committee", I guess. I just don't like "Steering committee".

**Comment by jbteves** [comment]
Honestly surprised we don't get this from GitHub. I'll make an issue.

File: newsletter-2021-02.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        Check them out!
+        User-facing documentation is available
+        <a href="https://tedana.readthedocs.io/en/latest/reporting.html">here</a>.
+        You can also see two example reports
+        <a href="https://me-ica.github.io/tedana-ohbm-2020/">here</a>.
+        </li>
+    <li>If no BOLD-like ICA components are found, tedana will now re-run the ICA step with a new seed.</li>
+    <li>Documentation has been improved both on readthedocs and inside of docstrings.</li>
+    <li>Added new documentation for multi-echo acquisition on Philips scanners <a href="https://tedana.readthedocs.io/en/latest/acquisition.html#philips">here</a>.</li>
+    <li>T2* values now have a nonzero floor.</li>
```
Comment (tsalo) [inline newsletter-2021-02.html]:
```suggestion
    <li>T2* values now have a nonzero floor to prevent divide-by-zero errors during optimal combination.</li>
```

I'd move this to Bug Fixes TBH.

File: newsletter-2021-02.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    <li>When opening an issue, there is now a neurostars link for more general questions.</li>
+    <li>Added variance-explained method for PCA (only mdl supported).</li>
```
Comment (tsalo) [inline newsletter-2021-02.html]:
```suggestion
    <li>Added variance-explained method for PCA.</li>
```
This new method is unrelated to the other ones.

File: newsletter-2021-02.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        User-facing documentation is available
+        <a href="https://tedana.readthedocs.io/en/latest/reporting.html">here</a>.
+        You can also see two example reports
+        <a href="https://me-ica.github.io/tedana-ohbm-2020/">here</a>.
+        </li>
+    <li>If no BOLD-like ICA components are found, tedana will now re-run the ICA step with a new seed.</li>
+    <li>Documentation has been improved both on readthedocs and inside of docstrings.</li>
+    <li>Added new documentation for multi-echo acquisition on Philips scanners <a href="https://tedana.readthedocs.io/en/latest/acquisition.html#philips">here</a>.</li>
+    <li>T2* values now have a nonzero floor.</li>
+    <li>When opening an issue, there is now a neurostars link for more general questions.</li>
```
Comment (tsalo) [inline newsletter-2021-02.html]:
```suggestion
    <li>When opening an issue, there is now a NeuroStars link for more general questions.</li>
```

**Review by tsalo — CHANGES_REQUESTED** [review]
Some minor things. In Chrome the commit history shows up as a paragraph, rather than a list. You might want line breaks or `<li>`.

EDIT: Also, is it worth mentioning that the JOSS manuscript is now open for community input?

File: newsletter-2021-02.html (hunk 1 above)
Comment (jbteves) [reply newsletter-2021-02.html]: I could just remove the period. I missed that it's on its own line. Thoughts?

I don't do much HTML either, it's just what tinyletter uses for formatting.

**Review by tsalo — APPROVED** [review]
LGTM!

**Review by emdupre — APPROVED** [review]
I didn't open in browser (I trust @tsalo 's review on that), but the plaintext content looks good to me ! Thanks @jbteves  !!

**Review by eurunuela — APPROVED** [review]
Looks good on Brave and the content seems to be okay. Thank you @jbteves !
